web/text_layer_builder.js

```diff
@@ -187,107 +187,116 @@ class TextLayerBuilder {
     this.#selectionChangeAbortController = new AbortController();
     const { signal } = this.#selectionChangeAbortController;
 
+    const on = (target, event, handler) =>
```

Comment:
I'm not really sure how I feel about this helper function yet. On the one hand it makes the call sites a bit simpler, but on the other hand it does introduce some indirection for an otherwise fairly regular `addEventListener` call. What do others think about this?

Reply:
Agreed, my initial reaction was that it feels a bit "unnecessary" to do this (even more so when the patch was updated to add the `target`-parameter).

Reply:
I did it because I was disliking the way passing the third param to addEventListener caused it to be formatted when having multiple short event listeners one after the other, but I'm happy to revert.